.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,3 +23,9 @@ for inclusion in changelog.
 #### Licensing
 
 This contribution is under the terms of the Apache 2.0 License: Yes
+
+#### Before merging
+<!-- Run integration and end-to-end tests before merging -->
+- [ ] cluster-state-service end-to-end tests pass. Required setup details in cluster-state-service/internal/Readme.md (`cd cluster-state-service; make e2e-tests; cd ../`)
```

Comment:
`make e2e tests` will not pass without setting the right environment variables (`ECS_CLUSTER`, etc.). We should probably mention that or not provide the command at all. Same with the 2 commands below.